### PR TITLE
Reserve expert details

### LINKED ISSUE #10: [3pt] Create a Form for Reservation
- Create a form that adds a form for the reservation.
- When the reservation is done successfully, show it in my reservation page.
- To reserve an appointment, the user has to select a date and city (username and selected item are autofilled).
- Use the design based on the ["Book a vespa test-ride"](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005583) and add all necessary inputs.
- The user can also access the "Reserve" page from the navigation panel. In that case only username is autofilled.

### PR DESCRIPTION
**Pull request by Theodoraldo**
### In this PR, I :
- Created a form that adds reservation
- Created a functionality when the reservation is done successfully, it should be shown in my reservation page.
- built a functionality that user can also access the "Reserve" page from the navigation panel. In that case, only username is autofilled.

### CODE REVIEW

File: src/components/Reservations/ReservationForm.js
Comment (Mofuhidy) [inline src/components/Reservations/ReservationForm.js:1]: You did a great job implementing navigate to reservations after adding the expert from the details, but could you also add it here after adding it from the sidebar?

Comment (AnsarIbrahim) [reply src/components/Reservations/ReservationForm.js:1]: I already do that on my side so don't do

**Review by Mofuhidy — CHANGES_REQUESTED** [review]
# Changes required ♻️ 
----------------------------

Hi @Theodoraldo 
While you did a great job .. but there is something to check.

Screenshot 2023-11-07 at 10 36 32 AM


this messgae appears after reserving expert even from the details page

**Review by anita00001 — COMMENTED** [review]
## Hi @Theodoraldo 

You did a great job on designing the reservation page. However, I would appreciate it even more if you could implemented the following:
image

In the above image, is it possible to autofill the highlighted values in the form itself?

**Review by Mofuhidy — DISMISSED** [review]
Hi @Theodoraldo 👋 ,
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

Congratulations! 🎉

Highlight ✅

✅ Good implementing add reservations.

Cheers, and Happy coding!👏👏👏

**Review by Mofuhidy — APPROVED** [review]
Hi @Theodoraldo 👋 ,
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

Congratulations! 🎉

Highlight ✅

✅ Good implementing add reservations.
✅  All changes Implemented

Cheers, and Happy coding!👏👏👏

**Pull request merged**